src/Sparrow.Server/Platform/Pal.cs

```diff
@@ -45,28 +46,26 @@ static Pal()
             }
             else
             {
-                throw new NotSupportedException("Not supported platform - no Linux/OSX/Windows is detected ");
+                throw new NotSupportedException("Not supported platform - no Windows/OSX/Linux is detected ");
             }
 
-            try
+            var retries = 0;
+            while (true)
             {
-                var copy = true;
-                if (File.Exists(toFilename))
-                {
-                    var fromHash = FileHelper.CalculateHash(fromFilename);
-                    var toHash = FileHelper.CalculateHash(toFilename);
+                if (CopyPalFile(toFilename, fromFilename, out Exception e))
+                    break;
 
-                    copy = fromHash != toHash;
+                if (++retries < 10)
+                    continue;
+
+                if (e is IOException)
+                {
+                    throw new IOException(
+                        $"Cannot copy {fromFilename} to {toFilename}, " +
+                        $"make sure appropriate {toFilename} to your platform architecture exists in Raven.Server executable folder", e);
                 }
 
-                if (copy)
-                    File.Copy(fromFilename, toFilename, overwrite: true);
-            }
-            catch (IOException e)
-            {
-                throw new IOException(
-                    $"Cannot copy {fromFilename} to {toFilename}, make sure appropriate {toFilename} to your platform architecture exists in Raven.Server executable folder",
-                    e);
+                throw e;
```

Comment:
Here you lost the exception trace, `throw new InvalidOperationException('msg', e);`

Reply:
done